CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ Pull requests are welcome on GitHub at https://github.com/zendesk/abbreviato
 
 After merging your changes into master, cut a tag and push it immediately:
 
-1. Update the version by `bundle exec rake bump:patch` or `bundle exec rake bump:minor`.
+1. Update the version in `lib/abbreviato/version.rb`.
```

Comment:
I’ll rewrite this section in the next PR, with the publishing workflow onboarding.